image/templates/helm/stackrox-secured-cluster/internal/cluster-config.yaml.tpl

```diff
@@ -29,5 +29,8 @@ clusterConfig:
       disableBypass: {{ ._rox.admissionControl.dynamic.disableBypass }}
       enforceOnUpdates: {{ ._rox.admissionControl.dynamic.enforceOnUpdates }}
     registryOverride: {{ ._rox.registryOverride }}
+    collectorConfig:
```

Comment:
I think this is important for the project.
Re-reading the state from a secret, e.g. like in this case the helm-cluster-config, requires Sensor to restart. I assume this would lead to Collectors restart.